src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -138,7 +138,10 @@
   --#{$menu-toggle}--m-plain--PaddingInlineEnd: var(--pf-t--global--spacer--action--horizontal--plain--default);
   --#{$menu-toggle}--m-plain--Color: var(--pf-t--global--icon--color--regular);
   --#{$menu-toggle}--m-plain--BackgroundColor: var(--pf-t--global--background--color--action--plain--default);
-  --#{$menu-toggle}--m-plain--BorderColor: transparent;
+  --#{$menu-toggle}--m-plain--BorderColor: var(--pf-t--global--border--color--high-contrast);
+  --#{$menu-toggle}--m-plain--BorderWidth: var(--pf-t--global--border--width--action--default);
```

Comment:
small nit -- but adding action--_plain_--[default/hover/clicked] will give the correct styles here for the plain menu toggles!